### PR TITLE
Adding SIP-49

### PR DESCRIPTION
**Pull request by frankTheTank72**
Adding a new standard for LPs on the Signum Chain

### CODE REVIEW

File: SIP/sip-49.md [hunk 1]
```diff
@@ -0,0 +1,553 @@
...
+
+6. **Solving the Sandwich Problem:** <br>
+One of the key innovations that this SIP brings to the Signum ecosystem is the solution to the "sandwich problem." By ensuring that all users get the same price within a single block, this innovation eliminates the arbitrage opportunities and market inefficiencies that traders can exploit, thus creating a fair and transparent trading environment for all participants.
+
+In conclusion, the motivation behind introducing Liquidity Pool Smart Contracts as a standard on the Signum platform is driven by a combination of factors that collectively aim to strengthen the ecosystem. By addressing liquidity, reducing friction, empowering developers, enhancing security, promoting global accessibility, incentivizing growth, and solving the "sandwich problem," this SIP strives to make Signum an attractive hub for DeFi activities, fostering innovation and enriching the user experience.
+
+
+##  Java code for the SRC-45 contract
+The Java code for the staking standard looks as follows:
```
Comment (ipr0310) [inline SIP/sip-49.md:75]: Java code for the SRC-49 contract
Not 45

Comment (ipr0310) [inline SIP/sip-49.md:75]: The Java code for the liquidity pool standard looks as follows:|

Not `staking` standard

File: SIP/sip-49.md [hunk 2]
```diff
@@ -0,0 +1,553 @@
...
+       -   After processing the swaps, the code calculates platform and liquidity provider fees for both directions (X to Y and Y to X).
+    -   It records the current block's platform and liquidity provider fees, ensuring that the fees are collected and distributed fairly.
+4.  **Reserve and Price Updates:**
+       - After processing all swap transactions in the block, the code updates the reserve amounts for token X and token Y to reconcile any dust or revenue.
+    -   It stores the reserve levels and prices at the end of the block, maintaining an accurate record of the pool state.
+
+This sequence ensures that liquidity-related transactions are processed first, followed by swap transactions with careful consideration of fees, slippage, and pricing. The code maintains a fair and secure environment for liquidity providers and users participating in swaps within the pool.
+
+### Creation Costs
+The creation of a staking contract costs 3.50 Signa. To be paid with the deploy-contract transaction. After creating the smart contract, the creator needs to send a second transaction with 152 Signa - to initiate the LP token on the contract.
```
Comment (ipr0310) [inline SIP/sip-49.md:541]: Creation of liquidity pool contract
Not staking

**Review by ipr0310 — COMMENTED** [review]
After applying these little changes, LGTM

**Comment by frankTheTank72** [comment]
Done

**Review by ipr0310 — APPROVED** [review]

**Pull request merged**